LoopKitUI/Views/Settings Editors/CarbRatioScheduleEditor.swift

```diff
@@ -11,7 +11,7 @@ import HealthKit
 import LoopKit
 
 
-fileprivate extension HKUnit {
+public extension HKUnit {
```

Comment:
Where else is this being used?

Reply:
In `TherapySettingsView.swift`.  I'll do as @mpangburn says below and move it to an HKUnit extension